charts/argo-cd/values.yaml

```diff
@@ -17,6 +17,29 @@ global:
     tag: ""
     # -- If defined, a imagePullPolicy applied to all Argo CD deployments
     imagePullPolicy: IfNotPresent
+
+  # -- If defined, uses a Secret to pull an image from a private Docker registry or repository
+  imagePullSecrets: []
+
+  logging:
+    # -- Set the global logging format. One of: `text` or `json`
```

Comment:
Waiting for 2.4.x release - I've seen that upstream added extra flags so these components should be configurable via env, but now it's possible only via command line args.

Reply:
imo it would be weird if the global logsetting dont work on some components. I would prefer using them as cmd flags if you want to release this before upstream additions.